os/fs/smartfs/Kconfig

```diff
@@ -89,20 +89,6 @@ config SMARTFS_JOURNALING
                 minimize the area reserved for journaling, it is advised to keep
                 sector size small.
                 
```

Comment:
same. Keeping it in KConfig without supporting code, is not correct.

Reply:
Same as above.